src/control/cmd/daos/util.h

```diff
@@ -4,6 +4,8 @@
  * SPDX-License-Identifier: BSD-2-Clause-Patent
  */
 
+#define D_LOGFAC	DD_FAC(client)
+
```

Comment:
I've only ever seen these logging macros defined in C files. It's probably OK here since this header has a very limited use, but I'll leave it to others with a better understanding of the logging system to comment.

Also noticed you didn't fence off the file with #ifndef ... #endif to avoid its contents being accidentally imported more than once. Again, may not be strictly required for this one, but still good practice.

Reply:
Will add the fencing. I think the theory behind adding it to .c files elsewhere is that a given header could be included from different files, but in this case as you say it's only going to be used in the daos tool. If we needed to enable different logging facilities from different parts of the tool this could be a problem, but I don't think that's going to happen. Mostly just trying to cut down on boilerplate.